src/components/sbb-header/sbb-header.scss

```diff
@@ -19,14 +20,22 @@
   position: fixed;
   inset: 0 0 auto;
   background: var(--sbb-color-white-default);
-  transition: box-shadow var(--sbb-animation-duration-4x) var(--sbb-animation-easing);
   z-index: 10;
+  transform: translateY(var(--sbb-header-position));
```

Comment:
the exact position is needed in order to simulate the header attached to the top of the content, and then have a smooth transition to the initial position (when you scroll it down) and the hidden position (just outside the screen): it is gradually translated as the page is scrolled down, until it's totally out of sight and gets attached to the top

Reply:
My idea was that it is not always position fixed, but just turns position fixed completely outside the view and before it just scrolls with the normal content. But then at least in storybook we have the problem, that examples are not always in full width (but maybe solvable). 
My doubt in your solution is that during the range where the header is scrolling out of the window, it can flickr, as the offset is constantly updated and depends on scroll event handling and good machines :) . But maybe I can gain more infos later.

Reply:
On my iPhone (13) it's flickering and looking bad somehow. The solution of css.ch looking much smoother.

Reply:
In the story file you could use the flag layout: 'fullscreen' and then it takes full width.

```export default {
  parameters: {
    docs: {
      extractComponentDescription: () => readme,
    },
    layout: 'fullscreen',
  },
  title: 'components/sbb-footer',
};```